spec/support/rails.rb

```diff
@@ -11,3 +11,31 @@ class DummyRailsEnv
     end
   end
 end
+
+class DummyRails
+  module VERSION
+    MAJOR = 4
+  end
```

Comment:
Can we have a dummy app for testing Rails 6 as well?

Reply:
not sure how I feel about dummy apps

Reply:
do people like dummy apps?